hypha/apply/projects/models/project.py

```diff
@@ -516,6 +516,26 @@ def __str__(self):
         return str(self.label)
 
 
+class ProjectReminderFrequency(Orderable, ClusterableModel):
+    num_days = models.IntegerField()
+    page = ParentalKey("ProjectSettings", related_name="reminder_frequencies")
+
+    class FrequencyRelation(models.TextChoices):
+        BEFORE = "BE", _("Before")
+        AFTER = "AF", _("After")
+
+    relation = models.CharField(
+        max_length=2,
+        choices=FrequencyRelation.choices,
+        default=FrequencyRelation.BEFORE,
+    )
+
+    panels = [
+        FieldPanel("num_days", heading=_("Number of Days")),
+        FieldPanel("relation", heading=_("Relation to Project Due Date")),
```

Comment:
Title case in several places.

Reply:
Instead of "Project due date" I think "Report due date" makes it more clear. There is no place to set a "Project due date. We have `proposed_start` and `proposed_end` in the model but they are currently unused.